notebooks/src/covid-19.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>For the first sentence, I'd say, "In this notebook, we model some aspects of how the COVID-19 pandemic will play out in the US at the county level."</p><p> </p><p>(current version sounds like we're comparing different counties to each other, which we're not)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>